sensor_msgs/PointCloud2 cloud1
sensor_msgs/PointCloud2 cloud2
geometry_msgs/Transform initial_guess
---
geometry_msgs/Transform transform